<?xml version="1.0"?>
<launch>
  <node name="test_image_evaluator" pkg="cps2" type="test_evaluator" output="screen" />
</launch>
